hackathon_site/hardware/api_urls.py

```diff
@@ -1,14 +1,18 @@
 from django.urls import path
 from hardware import views
 
-
 app_name = "hardware"
 
 urlpatterns = [
-    path("hardware/", views.HardwareListView.as_view(), name="hardware-list",),
+    path("hardware/", views.HardwareListView.as_view(), name="hardware-list"),
     path("orders/", views.OrderListView.as_view(), name="order-list"),
     path("categories/", views.CategoryListView.as_view(), name="category-list"),
     path("incidents/", views.IncidentListView.as_view(), name="incident-list"),
+    path(
+        "hardware/incidents/<int:pk>/",
```

Comment:
This will create an api called `"api/hardware/hardware/incidents/<int:pk>"` so remove the hardware part, the url should be `"incidents/<int:pk>/"` 
always check swagger to verify the api is correct before pushing code